force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -170,9 +312,24 @@ public with sharing class FlowControlLoopsCollections {
             // If it is not the last iteration, add a semicolon to separate the strings
         }
         */
+        String newString='';
+       
+        for (Integer i = 1; i <= repeatCount; i++){
+
+            newString = newString + inputString;
+
+            if (i != repeatCount){
+                newString = newString + '; ';
+            }
+         //   else {
+          //      newString = newString +'';
+          //  }
+
```

Comment:
It's ok to have these when practicing but uneeded comments should be removed from your final product